tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/Trusts/Academies/AcademiesPageModelTests.cs

```diff
@@ -17,6 +18,13 @@ public class AcademiesPageModelTests
     private readonly Mock<ILogger<AcademiesPageModel>> _mockLogger = new();
     private readonly AcademiesPageModel _sut;
 
+    private readonly DataSourceServiceModel _giasDataSource =
+        new(Source.Gias, new DateTime(2025, 1, 1), UpdateFrequency.Daily);
+
+    private readonly DataSourceServiceModel _eesDataSource = new(Source.ExploreEducationStatistics,
+        new DateTime(2025, 1, 1),
```

Comment:
Lets use our date time provider as in the future we can look to mock this more effectively if needed